packages/web-components/fast-foundation/src/carousel/carousel.template.ts

```diff
@@ -0,0 +1,98 @@
+
+const playIcon: string = `<svg viewBox="0 0 16 16" width="16px" height="16px" xmlns="http://www.w3.org/2000/svg"><path d="M1 0V16L15.0083 8L1 0Z"></path></svg>`;
+
+const basicTemplate: ViewTemplate = html<Carousel>`
+    <div class="carousel-content" ${ref("basicContent")}>
+        <slot ${slotted({ property: "items", filter: elements() })}></slot>
+    </div>
+`;
+
+const tabbedTemplate: ViewTemplate = html<Carousel>`
+    <div class="carousel-content">
+        <div class="tablist" part="tablist" role="tablist" ${ref("tablistRef")}>
+            <slot class="tab" name="tab" part="tab" ${slotted("tabs")}></slot>
+        </div>
+        <div class="tabpanel" ${ref("tabPanelsContainerRef")}>
+            <slot name="tabpanel" part="tabpanel" ${slotted("tabpanels")}></slot>
+        </div>
+    </div>
+`;
+
+/**
+ * The template for the {@link @microsoft/fast-foundation#(Carousel:class)} component.
+ * @public
+ */
+export const CarouselTemplate = html<Carousel>`
+<template
+    role="group"
+    aria-roledescription="carousel"
+    ${ref("carousel")}
+>
+    <div
+        class="rotation-control"
+        aria-label="${x =>
+            x.paused
+                ? "start automatic slide rotation"
+                : "stop automatic slide rotation"}"
+        ${ref("rotationControl")}
+    >
+        <slot
+            name="rotation-control"
+            part="rotation-control"
+            ${slotted("rotationControlItem")}
+        >
+            <fast-button appearance="neutral">
+                ${x =>
+                    x.paused
+                        ? html`
+                              ${playIcon}
+                          `
+                        : html`
+                              ${pauseIcon}
+                          `}
+            </fast-button>
+        </slot>
+    </div>
+    <div 
+        class="previous-flipper flipper"
+        @click=${(x, c) => x.handleFlipperClick(-1, c.event as MouseEvent)}
+        @keypress=${(x, c) => x.handleFlipperKeypress(-1, c.event as KeyboardEvent)}
+    >
+        <slot name="previous-button" part="previous-button" ${slotted(
+            "previousFlipperSlottedItem"
+        )}>
+            <fast-flipper aria-label="previous slide" aria-hidden="${x =>
+                !x.basicPattern ? "true" : "false"}" direction=${
+    FlipperDirection.previous
+}
+                ${ref("previousFlipperDefault")}
+>
+        </slot>
+    </div>
+    <div
+        class="next-flipper flipper"
+        @click=${(x, c) => x.handleFlipperClick(1, c.event as MouseEvent)}
+        @keypress=${(x, c) => x.handleFlipperKeypress(1, c.event as KeyboardEvent)}
+    >
+        <slot name="next-button" part="next-button" ${slotted("nextFlipperSlottedItem")}>
+            <fast-flipper aria-label="next slide" aria-hidden="${x =>
```

Comment:
We don't want to bake strings into the template here because they can't be easily localized - you should make the labels property driven or find another mechanism to make the labels configurable